CONTRIBUTING.md

```diff
@@ -129,3 +129,24 @@ Please abstain from temporarily closing a pull request while revising the templa
 
 Once the reviewers have merged the pull request, our [build server](http://build.voidlinux.org) is automatically triggered and builds
 all packages in the pull request for all supported platforms. Upon completion, the packages are available to all Void Linux users.
+
+## Testing Pull Requests
+
+Testing new package submissions and updates is always helpful, and is a great way to get started with contributing.
+First, [clone the repository](https://github.com/void-linux/void-packages#quick-start) if you haven't done so already.
+Then check out the pull request, either with `github-cli`:
+
+    $ gh pr checkout <number>
+
+Or manually:
+
+    $ git fetch origin pull/<number>/head:<branch-name>
+    $ git checkout <branch-name>
+
+If your local void-packages repository is cloned from your fork, you may need to add the main repository as a remote first:
+
+    $ git remote add upstream https://github.com/void-linux/void-packages.git
```

Comment:
This should go before fetching to preserve time order, and before `gh` if applies there.

Reply:
tested and `gh` is able to do it without adding any remotes from either the fork clone or base repo clone